db/getMetaData.js

```diff
@@ -12,21 +12,21 @@ const getMetaData = (productId, callback) => {
       metaData.ratings = {};
       if (res.rows.length) {
         res.rows.forEach((rat) => {
-          metaData.ratings[rat.rating] = rat.count;
+          metaData.ratings[rat.rating] = parseInt(rat.count, 10);
```

Comment:
For readability, you might want to make this variable "rating" instead of "rat".